.github/workflows/windows-builds.yml

```diff
@@ -0,0 +1,212 @@
+name: 'Build on Windows'
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  win64:
+    name: 'Windows 64-bit'
+    runs-on: [windows-latest]
+    env:
+      QT_VERSION: '5.12.9'
+      CMAKE_GENERATOR: "Visual Studio 16 2019"
+    steps:
+      - name: 'Add msbuild to PATH'
+        uses: microsoft/setup-msbuild@v1.0.2
+      - name: 'Checkout'
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: 'Get Cockatrice git info'
+        shell: bash
+        working-directory: ${{ github.workspace }}
+        run: |
+          git fetch --prune --unshallow
+          echo "GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)" >> $GITHUB_ENV
+          echo "GIT_HASH=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
+          echo "GIT_TAG=$(git describe --tags --abbrev=0)" >> $GITHUB_ENV
+      - name: Restore Qt 64-bit from cache
+        id: cache-qt
+        uses: actions/cache@v2
+        with:
+          path: |
+            ${{ runner.workspace }}/Qt64
+          key: ${{ runner.os }}-QtCache-64bit
+      - name: 'Install 64-bit Qt'
+        uses: jurplel/install-qt-action@v2
+        with:
+          cached: ${{ steps.cache-qt.outputs.cache-hit }}
+          version: '${{ env.QT_VERSION }}'
+          arch: 'win64_msvc2017_64'
+          dir: ${{ runner.workspace }}/Qt64
+      - name: 'Restore from cache and run vcpkg'
+        uses: lukka/run-vcpkg@v5
+        with:
+          vcpkgArguments: '@${{ github.workspace }}/vcpkg.txt'
+          vcpkgDirectory: '${{ github.workspace }}/vcpkg'
+          appendedCacheKey: ${{ hashFiles(env.vcpkgResponseFile) }}
+      - name: 'Configure Cockatrice 64-bit'
+        working-directory: ${{ github.workspace }}
+        run: |
+          New-Item build64 -type directory -force
+          cd .\build64
```

Comment:
```suggestion
          cd build64
```

Reply:
This suggestion is b/c of https://github.com/Cockatrice/Cockatrice/pull/4193/files#diff-ca5dd169a0cae96b54f57acfa81f594ba1fde1358a8b8488a5aaeb619e985736R71